client/errors.go

```diff
@@ -153,13 +153,14 @@ func isCodeThrottle(code string) bool {
 }
 
 var (
-	requestIdRegex = regexp.MustCompile(`\s(RequestID:|request id:)\s[A-Za-z0-9-]+`)
+	requestIdRegex = regexp.MustCompile(`\s(RequestID:|request id:|Request ID:)\s[A-Za-z0-9-]+`)
```

Comment:
nit: maybe make the regexes case insensitive?

Reply:
So `ReQuEsT iD` can also pass? It's too much I think 😂

Reply:
but I fixed the regex for you